Snowflake.Data.Tests/UnitTests/Configuration/EasyLoggingConfigFinderTest.cs

```diff
@@ -126,6 +126,21 @@ public void TestThatReturnsNullIfNoWayOfGettingTheFile()
             Assert.IsNull(filePath);
         }
 
+        [Test]
+        public void TestThatReturnsNullIfDirectoryBasedSearchFailsWithUnexpectedError()
```

Comment:
Is it a correct level of testing? I would expect more like integration test.

Reply:
I think there is no need for that since all I changed is covered only in this class.